CodeStarterCamp_Week3/coffeeShop.swift

```diff
@@ -12,34 +12,46 @@ enum Coffee: String {
     
     var name: String {
         switch self {
-        case .americano: return "Americano"
-        case .iceAmericano: return "Ice Americano"
-        case .cafeLatte: return "Cafe Latte"
-        case .hotChoco: return "Hot Choco"
-        case .vanillaLatte: return "Vanilla Latte"
+        case .americano: return "아메리카노"
+        case .iceAmericano: return "아이스아메리카노"
+        case .cafeLatte: return "카페라떼"
+        case .hotChoco: return "핫초쿄"
+        case .vanillaLatte: return "바닐라라떼"
         }
     }
 }
 
 class CoffeeShop {
     var salesAmount: Int = 0
     var menuPan: [Coffee: Int] = [:]
-    var pickUpTable: [Coffee] = []
     var barista: Person
+    var pickUpTable: [[Coffee: String]] = [[:]] {
```

Comment:
`pickUpTable`의 타입을 딕셔너리로 바꿔주셨군요!
key, value를 활용해서 주문한 사람의 이름과 메뉴를 같이 관리하는 방법 좋습니다 👍

하지만 배열 안에서 딕셔너리를 활용하시는 것은 살짝 복잡해보일 수 있을 것 같습니다.
딕셔너리 / 배열 중 하나를 선택해서 사용해주시면 좋을 것 같아요! 🤔
이럴 때는 튜플타입을 활용해보시는 것도 좋겠습니다 😊
```suggestion
    var pickUpTable: [(Coffee, String)] = [] {
```